bliss/encoder/metrics.py

```diff
@@ -372,18 +372,18 @@ def plot(self):
 class SourceTypeAccuracy(FilterMetric):
     def __init__(
         self,
-        flux_bin_cutoffs: list,
+        mag_bin_cutoffs: list,
```

Comment:
In a recent PR, @YicunDuanUMich has changed this variable name to `bin_cutoffs`, and added another variable to specify the type of bin. The type can be either "flux" or "mag". Perhaps that change will help here

Reply:
Updated all the metrics to take `bin_cutoffs`, `ref_band`, and `bin_type`